idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -0,0 +1,659 @@
+        self.c0 = Var(self.flowsheet().config.time,
+                      initialize=260,
+                      doc="Speed of Sound in Inlet Gas Stream",
+                      bounds=(0, 4000),
+                      units=pyunits.m / pyunits.s)
+        self.delta_enth_polytropic = Var(self.flowsheet().config.time,
+                                         initialize=2548.5,
+                                         doc="Polytropic Enthalpy Change",
+                                         units=pyunits.J / pyunits.mol,
+                                         within=NonNegativeReals)
+        self.impeller_work_coeff = Var(self.flowsheet().config.time,
+                                       initialize=0.6993,
+                                       doc="Impeller Work Coefficient",
+                                       bounds=(0.001, 1))
+        self.mu_p_v = Var(self.flowsheet().config.time,
+                          initialize=0.60,
+                          doc="Polytropic Head Coefficient Vaned Diffuser",
+                          bounds=(0.001, 1))
+        self.mu_p = Var(self.flowsheet().config.time,
+                        initialize=0.60,
+                        doc="Polytropic Head Coefficient",
+                        bounds=(0.001, 1))
+        self.eff_p_v = Var(self.flowsheet().config.time,
+                           initialize=0.85,
+                           doc="Polytropic Efficiency Vaned Diffuser",
+                           bounds=(0.001, 1))
+        self.eff_p = Var(self.flowsheet().config.time,
+                         initialize=0.85,
+                         doc="Polytropic Efficiency",
+                         bounds=(0.001, 1.0))
+        self.Ma = Var(self.flowsheet().config.time,
+                      initialize=0.1,
+                      doc='Rotational Mach Number',
+                      bounds=(0, 5))
+        self.kappaT = Var(self.flowsheet().config.time,
+                          initialize=5.0e-7, units=1 / pyunits.Pa)
+        self.Ma_max = Var(self.flowsheet().config.time,
+                          initialize=0.1,
+                          doc='Rotational Mach Number Upper limit',
+                          bounds=(0, 5))
+        # dynamic related variables
+        self.psi_3 = Var(self.flowsheet().config.time,
+                         initialize=0.01,
+                         doc="Dimensionless Exit Flow Coefficient",
+                         within=NonNegativeReals,
+                         units=pyunits.dimensionless)
+        self.psi_s = Var(self.flowsheet().config.time,
+                         initialize=0.01,
+                         doc="Dimensionless Isentropic Head Coefficient",
+                         within=NonNegativeReals,
+                         units=pyunits.dimensionless)
+        ########################################################
+        # Declare parameters for the model
+        self.k_v = Param(default=1.2766,
```

Comment:
It is probably better just to use all Var rather than Param, and fix the variables.  If you know the parameter values, it doesn't mater, but if you want to use parameter estimation, you'll want these to be variables.  I don't think I would uses parameters unless it's a value you really know, and would never try to estimate for some kind of data.